python/ray/tune/stopper.py

```diff
@@ -107,9 +116,23 @@ def __call__(self, trial_id, result):
             self._top_values = sorted(self._top_values)[:self._top]
         else:
             self._top_values = sorted(self._top_values)[-self._top:]
+
+        # If the current iteration has to stop
+        if self._stop():
+            # we increment the total counter of iterations
+            self._iterations += 1
+        else:
+            # otherwise we reset the counter
+            self._iterations = 0
+
+        # and then call the method that re-executes
+        # the checks, including the iterations.
         return self.stop_all()
 
-    def stop_all(self):
-        """Return whether to stop and prevent trials from starting."""
+    def _stop(self):
```

Comment:
nit (clarity) - Maybe let's rename this to "has_plateaued"

Reply:
Do you mean the method `_stop` or `stop_all`? The second one is a default method, but for the first one by all means!